sphinx/source/docs/releases/0.12.6.rst

```diff
@@ -58,3 +58,19 @@ Old                            New
 Since these attributes may be present in user code (e.g CustomJS callbacks
 or extensions), the old names will continue to work for some time, with a
 deprecation warning in the JS console.
+
+New signaling API
+~~~~~~~~~~~~~~~~~
+
+Previously we used Backbone events for communication between bokehjs' objects.
+This was changed to a new type-safe API. This change mostly affects contributors
+to bokehjs and writers of extensions.
+
+===================================== ==============================================
+Old                                   New
+===================================== ==============================================
+
+``@listenTo(obj, "change", fn)``      ``@connectTo(obj.change, fn)``
+``@listenTo(obj, "change:some", fn)`` ``@connectTo(obj.properties.some.change, fn)``
+``obj.trigger("change", arg)``        ``obj.change.emit(arg)``
+``obj.trigger("change:some", arg)``   ``obj.properties.some.change.emit(arg)``
```

Comment:
`foo` would also be fine :) Do you mind making those two small changes? (attr name in notes and `connect_to`) ? Then we can merge and get a dev build out